iib/web/models.py

```diff
@@ -373,6 +374,11 @@ def _state_to_json(state):
             states = list(reversed(states))
             rv['state_history'] = states
             latest_state = states[0]
+            if current_app.config['IIB_REQUEST_LOGS_DIR']:
+                rv['logs'] = {
+                    'expiration': self.logs_expiration.isoformat() + 'Z',
+                    'url': url_for('.get_build_logs', request_id=self.id, _external=True),
```

Comment:
Should we hard code the api version in the model? This seems to work without specifying it which makes me wonder that it will use whichever api version is being used in the request.

Reply:
If you can confirm that, then we can keep it as is.